codecoverage.py

```diff
@@ -91,10 +91,15 @@ def generate_info(grcov_path):
         else:
             ordered_files.append("ccov-artifacts/" + fname)
 
+    # Assume we're on a one-click loaner.
+    mod_env = os.environ.copy()
+    if os.path.isdir('/home/worker/workspace/build/src/gcc/bin'):
```

Comment:
nit: could put the dir path in a variable, like `one_click_loaner_gcc` as an alternative to just documenting this case with a comment